### ISSUE TITLE
Appimage creates /compatibilitytools.d in ~/.local/share/Steam instead of ~/.steam/root

### ISSUE THREAD

**Issue by LinuxJoeUser** [DavidoTek/ProtonUp-Qt#173]
**Describe the bug**  
ProtonUP-QT (on PUQ  2.5 and 2.7.7) create `~/.local/share/Steam/compatibilitytools.d/` where the Steam client doesn't look for PUQ's downloaded files , instead of in `~/.steam/root/compatibilitytools.d`  where Steam does see the files and they don't appear in Steam's "Force compatibility tools" drop down.

**To Reproduce**  
Run the .appimage on Xubuntu 22.04, I suppose.

**Expected behavior**  
Steam client would see and scan /compatibilitytools.d and make those files available to user in Steam client's "Force compatibility tool" drop down.

**Desktop (please complete the following information):**  
 - **Platform**: PC
 - **System**: Xubuntu 22.04
 - **Version**: For example ProtonUp-Qt 2.7.7
 - **How did you install ProtonUp-Qt?**: AppImage

**Additional context**  
- **CPU**: Ryzen 9 3900x
- **GPU**: Radeon rx 6600 xt
- **mesa**: 22.2.5 from kisak/turtle repo
- **kernal**: 5.15.0-58-generic
- **Steam**: Latest, stable
- **Steam Source**: installed using Steam Installer from Xubuntu's repos.

**Terminal output**  
Interesting to note that the Appimage isn't seen by bash, either, so launching it that way doesn't work.  Other appimages can be run from the cli (e.g., sc-controller, bitwarden, antimicro).

```$ ./ProtonUp-Qt-2.7.7-x86_64.Appimage
bash: ./ProtonUp-Qt-2.7.7-x86_64.Appimage: No such file or directory
$ ls
AntiMicroX-x86_64-3.2.2.AppImage    Leonflix.AppImage
balenaEtcher-1.10.2-x64.appimage    Linphone-4.2.5.AppImage
balenaEtcher-1.5.121-x64.AppImage   LosslessCut-linux-x86_64.AppImage
Bitwarden-2022.6.2-x86_64.AppImage  MyPaint-v2.0.0.AppImage
cryptomator-1.5.5-x86_64.AppImage   ProtonUp-Qt-2.7.7-x86_64.AppImage
Deskreen-2.0.3.AppImage             scc-contents
freetube_0.17.0_amd64.AppImage      sc_controller-0.4.8.9.glibc-x86_64.AppImage
Heroic-2.5.2.AppImage               scc_versions
How-to-aunch-appimage-from-cli.txt  SMPlayer-21.10.0-x86_64.AppImage
$ ./sc_controller-0.4.8.9.glibc-x86_64.AppImage
pidfile /home/gw/.config/scc/daemon.pid does not exist. Daemon not running?
W ContImage     Area for button DOTS not found
W ContImage     Area for button DOTS not found
``` 

Let me know if I can provide more info, and thanks for all your work on ProtonUp-QT, it's a well-designed, time-saving tool.  Keep up the good work!

**Comment by DavidoTek** [comment]
It looks like there are multiple Steam installations on the same system. Normally, `~/.steam/root` is just a link to `~/.local/share/Steam`.
That could be caused by an older Steam installation where the directory was different.

Maybe try to check which folder is the correct one of the two folders and delete the other one if it's not needed anymore.

Please let me know if this solved the issue.

**Comment by sonic2kk** [comment]
I wonder if this is resolved with the changes made to how we select the Steam install directory?

**Comment by DavidoTek** [comment]
> I wonder if this is resolved with the changes made to how we select the Steam install directory?

In theory, commit https://github.com/DavidoTek/ProtonUp-Qt/commit/95b2e6b229564dc05dfc53e0ffd52a8b5805dab8 should cover this issue as it is less likely that the old folder contains both `config.vdf` and `libraryfolder.vdf`. The commit is included with version 2.8.1.

In case they still remain in the wrong folder from an older installation, the issue would still persist. That would need a different way of checking for the correct folder, e.g. prioritize `~/.steam/root` as it is the most common and correct folder.
